coremltools/converters/mil/mil/ops/defs/linear.py

```diff
@@ -91,52 +83,52 @@ class matmul(Operation):
     * ``Matmul(x, y).shape == (4, 2)``
     
     If ``x`` is 1-D, and ``y`` is N-D where ``N >= 2``, ``x`` is first promoted to
-    matrix ``xm``by prepending a ``1`` to its dimension, and the resulting ``xm`` is
-    broadcast to ``y`` following Rule 2. Remove the inserted dimension afterwards.
+    matrix ``xm`` by prepending a ``1`` to its dimension, and the resulting ``xm`` is
+    broadcast to ``y`` following Rule 2 above. After this, remove the inserted dimension.
     For example:
     
     * ``x.shape == (4)``
     * ``y.shape == (10, 4, 3)``
     * ``xm.shape == (1, 4)``
     * ``Matmul(xm, y).shape == (10, 1, 3)``
     * Removing the inserted dimension results in ``Matmul(x, y).shape == (10, 3)``.
-    
-    Note: ``xm`` and ``Matmul(xm, y)`` are for illustration only.
+    * Note: ``xm`` and ``Matmul(xm, y)`` are for illustration only.
     
     If ``x`` is N-D where ``N >= 2``, and ``y`` is 1-D, ``y`` is first promoted to
     matrix ``ym`` by appending a ``1`` to its dimension, and the resulting ``ym`` is
-    broadcast to ``x`` following Rule 2. Remove the inserted dimension afterwards.
+    broadcast to ``x`` following Rule 2 above. After this, remove the inserted dimension.
     For example:
     
     * ``x.shape == (10, 3, 4)``
     * ``y.shape == (4,)``
     * ``ym.shape == (4, 1)``
     * ``Matmul(x, ym).shape == (10, 3, 1)``
```

Comment:
Maybe lower case `matmul(...)`? Or `mb.matmul(...)` matching other examples such as 2nd to the last example in https://coremltools.readme.io/docs/composite-operators.

Reply:
@aseemw - Can you comment on this? The source document [MIL Core Ops](https://quip-apple.com/1mNfAW4JhWR9#PDe9CATH3Ho) uses Matmul(x, ym).

Reply:
I think we can standardize using lower case for all the ops.

Reply:
@1duo I didn't "dismiss" your "stale review" -- I don't know what that comment means. I did make the changes from "Matmul" to "matmul" (lower case) throughout. Thanks!

Reply:
No worry. It clears out stale approvals whenever a push happens